node/tests/RedisClient.test.ts

```diff
@@ -365,6 +365,92 @@ describe("GlideClient", () => {
         TIMEOUT,
     );
 
+    it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
+        "copy with DB test_%p",
+        async (protocol) => {
+            if (cluster.checkIfServerVersionLessThan("6.2.0")) return;
+
+            const client = await GlideClient.createClient(
+                getClientConfigurationOption(cluster.getAddresses(), protocol),
+            );
+
+            const source = `{key}-${uuidv4()}`;
+            const destination = `{key}-${uuidv4()}`;
+            const value1 = uuidv4();
+            const value2 = uuidv4();
+            const index0 = 0;
+            const index1 = 1;
+            const index2 = 2;
+
+            try {
+                // neither key exists
+                checkSimple(
```

Comment:
I think checkSimple was created to compare strings, so even though it still passes  with non-strings, I think we should be using the regular `expect` when the return value is not a string or an array that contains strings. The error message is also clearer when we use expect instead of checkSimple

Reply:
If I use `expect`, the check will fail. That's why I use `checkSimple` instead.

Reply:
What's the error message when it fails? I'm pretty sure this should pass if using expect instead of checkSimple, since I think checkSimple was introduced specifically for comparing results that contain string values and this does not return a string